server/package.json

```
@@ -0,0 +1,74 @@
+{
+  "name": "@pass-culture/web-server",
+  "description": "Web server proxy for pass-culture-app-native",
+  "private": true,
+  "license": "UNLICENSED",
+  "main": "build/src/index.js",
+  "author": {
+    "email": "support@passculture.app",
+    "url": "https://pass.culture.fr",
+    "name": "pass Culture",
+    "twitter": "@pass_Culture"
+  },
+  "contributors": [
+    "Dimitri Kopriwa <dimitri.kopriwa@passculture.app>"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/pass-culture/pass-culture-app-native.git"
+  },
+  "engines": {
+    "node": "16"
+  },
+  "scripts": {
+    "build": "tsc --build",
+    "predeploy": "yarn build",
+    "deploy": "gcloud beta app deploy --quiet",
+    "deploy:testing": "cat .env.testing > .env && sed -i.back -e '/web-server-testing/! s/web-server/web-server-testing/g' app.yaml && yarn deploy",
+    "deploy:staging": "cat .env.staging > .env  && sed -i.back -e '/web-server-staging/! s/web-server/web-server-staging/g' app.yaml && yarn deploy",
+    "deploy:integration": "cat .env.integration > .env && sed -i.back -e '/web-server-integration/! s/web-server/web-server-integration/g' app.yaml && yarn deploy",
+    "deploy:production": "cat .env.production > .env && sed -i.back -e '/web-server-production/! s/web-server/web-server-production/g' app.yaml && yarn deploy",
+    "dev": "ENV=testing node-dev src/index.ts",
+    "start": "node build/src/index.js",
+    "test": "yarn test:lint && yarn test:types && yarn test:unit",
+    "test:lint": "eslint . --ext .js,.ts --cache",
+    "test:types": "tsc --noEmit",
+    "test:unit": "TZ=UTC JEST=true jest --forceExit",
+    "test:unit:ci": "TZ=UTC yarn test:unit --maxWorkers=2 --logHeapUsage --forceExit --coverage",
```

le script `test:unit:ci` rappelle le script `test:unit`

il y a pas besoin de répéter `TZ=UTC` et `--forceExit`

```suggestion
    "test:unit:ci": "yarn test:unit --maxWorkers=2 --logHeapUsage --coverage",
```